bin/happo-ci

```diff
@@ -82,6 +86,11 @@ run-happo "$CURRENT_SHA" true
 if ! "$HAPPO_COMMAND" has-report "$PREVIOUS_SHA"; then
   echo "No previous report found for ${PREVIOUS_SHA}. Generating one..."
   run-happo "$PREVIOUS_SHA" false
+  
+  # Restore git and node_modules to their original state, for anything that
+  # might be run after this script.
+  git checkout "$CURRENT_SHA"
```

Comment:
This will have to be `${HAPPO_GIT_COMMAND} checkout "$CURRENT_SHA"`. The happo-ci-test relies on this to override the git command. 

Also, perhaps we should be a little more aggressive to prevent it from failing: `${HAPPO_GIT_COMMAND} checkout --force --quiet "$CURRENT_SHA"`.